README.md

```diff
@@ -1,14 +1,18 @@
 # Cockpit Files
 
+This is the [Cockpit](https://cockpit-project.org/) user interface for managing
+files.
+
+
 # Development dependencies
 
 On Debian/Ubuntu:
 
-    $ sudo apt install gettext nodejs npm make
+    sudo apt install gettext nodejs npm make
 
 On Fedora:
 
-    $ sudo dnf install gettext nodejs npm make
+    sudo dnf install gettext nodejs npm make
```

Comment:
Nit pick, we should fix this at the source https://github.com/cockpit-project/starter-kit/?tab=readme-ov-file#development-dependencies

Reply:
Right, I sent https://github.com/cockpit-project/starter-kit/pull/912